infrastructure/db/factories/testDb.js

```diff
@@ -3,19 +3,14 @@ const { models, initDb } = require("..");
 
 const testDatabase = async () => {
   await initDb();
-  models.teammember.create({ emailAddress : faker.internet.email(), password : faker.internet.password() })
-    .then(user => {
-      models.board.create({label : 'FactoryBoard'}).then(board => {
-      //board.setTeammembers(user, {});
-      //user.addBoard(board, {});
-      //only use one of them. they do the same => add relation to the pivot table
-      models.group.create({ label : 'groupFactory', boardId: board.id })
-      .then(group => {
-          models.task.create({ label : 'taskFactory', status : 'unfinished',
-            metadata : {}, groupId: group.id, teammemberId: user.id, teammeberId: user.id })
-      });
-    });
-  });
+  const member = await models.teammember.create({ emailAddress : faker.internet.email(), password : faker.internet.password() });
+  const board = await models.board.create({label : 'FactoryBoard'});
+  //board.setTeammembers(member, {});
+  //member.addBoard(board, {});
+  //only use one of them. they do the same => add relation to the pivot table
+  const group = await models.group.create({ label : 'groupFactory', boardId: board.id });
+  models.task.create({ label : 'taskFactory', status : 'unfinished',
```

Comment:
here on line 12, you are not using `await` which means that this function will only be thrown to the event loop without waiting for the execution to finish. Is this the intended behaviour?